### PR TITLE
Blockbase + children: increased minor version 

### PR DESCRIPTION
**Pull request by MaggieCabrera**
After adding the color customizations functionality we should increase the minor version of these themes. I updated the changelog accordingly.

### CODE REVIEW

File: blockbase/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "blockbase",
-  "version": "1.0.1",
+  "version": "1.1.2",
```
Comment (scruffian) [inline blockbase/package.json]: Why 1.1.2, not 1.1.0?

Comment (MaggieCabrera) [reply blockbase/package.json]: because the package.json wasn't up to date, the stylesheet was 1.0.2

Comment (scruffian) [reply blockbase/package.json]: right but i think we should go back to 0 in the third position when the second position increases...

Comment (MaggieCabrera) [reply blockbase/package.json]: oh yeah, I don't know what I was thinking! Let me change it

**Review by scruffian — APPROVED** [review]
LGTM

**Pull request merged**